windows/wintools_complete_install_verify/verify_vmtools.yml

```diff
@@ -41,6 +41,12 @@
     msg: "Service '{{ item.key }}' status is: {{ item.value }}"
   with_dict: "{{ vmtools_service_dict }}"
 
+- name: "Check 4 services in the list"
+  ansible.builtin.assert:
+    that:
+      - vmtools_service_dict | length == 4
+    fail_msg: "Expect 4 VMware services listed after VMware Tools install, while there are '{{ vmtools_service_dict | length }}' got."
```

Comment:
Here can also print `vmtools_service_dict.keys()` in `fail_msg` to help triage.

Reply:
Added, thanks.